README.md

```diff
@@ -1,75 +1,159 @@
 # generate-build-provenance
 
-Given a subject (either a file path or name/digest pair), this action creates,
-signs, and uploads a build provenance attestation using [Sigstore][1].
+GitHub Action to create, sign and upload a build provenance attestation for
+artifacts built as part of a workflow.
+
+The generated attestation is a [SLSA provenance][2] document which captures
```

Comment:
i would say dump this _after_ the usage section just cos it adds a lot to scroll before the main useful bit.